core-rust/state-manager/src/staging/cache.rs

```diff
@@ -312,6 +312,57 @@ impl<'s, S: SubstateDatabase> SubstateDatabase for StagedStore<'s, S> {
         }
     }
 
+    fn list_entries_from(
```

Comment:
Oh - I guess it relates to an addition to the `SubstateDatabase` trait? I'm surprised this wasn't required before?

Reply:
@dhedey yeah, this method was added by me at https://github.com/radixdlt/radixdlt-scrypto/pull/1648 in order to address this Browse API performance TODO: https://github.com/radixdlt/babylon-node/pull/741/files#diff-148a2d8426fe568c7f0cab8adda124b9958ea5f3bc23e41c490d566bc8ffe103R822 in near future; and yeah, Node's DB simply has to impl it 👍 
_(and it was not required before, because in all use cases [before Browse API] we always iterated the entire partition from its beginning)_

@talekhinezh thanks for pulling this here; can I just recommend deleting all the `list_entries()` impls plz? _(the trait right now provides the default impl for it, which is more than appropriate for all cases that I can imagine)_

Reply:
Ah, I don't think there was a default implementation when I first pulled in the new radixdlt scrypto. Will remove it now.